api/client/run.go

```diff
@@ -254,6 +255,16 @@ func (cli *DockerCli) CmdRun(args ...string) error {
 
 	// Attached mode
 	if *flAutoRemove {
+		// Warn user if they detached us
+		js, err := cli.client.ContainerInspect(createResponse.ID)
+		if err != nil {
+			return runStartContainerErr(err)
+		}
+		if js.State.Running == true || js.State.Paused == true {
+			fmt.Fprintf(cli.out, "\ndocker: detached from %s! Waiting for it to terminate in order to uphold \"--rm\".\n",
```

Comment:
please, remove the initial `\ndocker: ` from the message. Also, please, no exclamations, we don't want to be rude and yell at people :grin: 

```go
fmt.Fprintf(cli.out, "Detached from %s. Waiting for it to terminate in order to uphold \"--rm\"\n",
```